perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -560,6 +567,10 @@ sub inspect : Private {
                     anonymous => 0,
                     %update_params,
                 } );
+                if ( my $fileid = $c->stash->{upload_fileid} ) {
+                    $update->photo($fileid);
+                    $update->update;
```

Comment:
I don't think this needs to be a separate db call, you could put it in the main add call, e.g. `photo => $c->stash->{upload_fileid} || undef,` or similar.